install_config/install/prerequisites.adoc

```diff
@@ -640,6 +640,18 @@ SELINUX=enforcing
 SELINUXTYPE=targeted
 ----
 
+[IMPORTANT]
```

Comment:
@sferich888 I didn't think this worked with the Important box that's currently there, so I created a new one incorporating it into the intro para and. WDYT?

I also got info from Eduardo (BZ reporter) in the BZ and I'll add some more info when he gets back to me.